### PR TITLE
Add a contribute.json file

### LINKED ISSUE #472: Add a contribute.json file to the repository
See https://www.contributejson.org/

### PR DESCRIPTION
**Pull request by magopian**
Fixes #472

### CODE REVIEW

File: contribute.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Kinto",
+  "description": "Kinto is a minimalist JSON storage service with synchronisation and sharing abilities.",
+  "repository": {
+    "url": "https://github.com/Kinto/kinto",
+    "license": "MPL2",
```
Comment (leplatrem) [inline contribute.json]: Apache actually ;)

**Comment by leplatrem** [comment]
r+wc :)

File: contribute.json (hunk 1 above)
Comment (magopian) [inline contribute.json]: ah, you're right ;)

**Comment by magopian** [comment]
@leplatrem updated the license field, thanks, is that ok?

**Comment by leplatrem** [comment]
> is that ok?

yup!

you can merge IMO

File: contribute.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "list": "https://github.com/Kinto/kinto/issues",
+    "report": "https://github.com/Kinto/kinto/issues/new"
+  },
+  "urls": {
+    "dev": "https://kinto.dev.mozaws.net/v1/"
+  },
+  "keywords": [
+    "JSON",
+    "Python",
+    "Pyramid"
```
Comment (almet) [inline contribute.json]: "REST", "API" ?

Comment (almet) [inline contribute.json]: (We might want to have a few more keywords here in order to have people discover us by the use of these, maybe) — Also thinking about "database".

**Pull request merged**